integration/config/config_test.go

```diff
@@ -327,3 +328,27 @@ func waitAndAssert(t *testing.T, timeout time.Duration, f func(*testing.T) bool)
 		time.Sleep(100 * time.Millisecond)
 	}
 }
+
+func TestConfigInspect(t *testing.T) {
+	skip.If(t, testEnv.DaemonInfo.OSType != "linux")
```

Comment:
Why is this skipped on Windows?

Reply:
@thaJeztah Both config and secret currently works with swarm so it is skipped on on-linux case.

Reply:
oh, right, had my wires crossed lol